filter.js

```diff
@@ -38,9 +41,9 @@ function Filter(inputTree, options) {
   this._destFilePathCache = Object.create(null);
 }
 
-Filter.prototype.rebuild = function() {
+Filter.prototype.build = function build() {
   var self = this;
-  var srcDir = this.inputPath;
+  var srcDir = this.inputPaths[0];
```

Comment:
It does guarantee there's exactly one object passed in, but that object may not be a broccoli tree (if the caller did in fact accidentally pass in an array for the first argument).